lib/linguist/languages.yml

```diff
@@ -2543,6 +2543,15 @@ MiniD:
   tm_scope: none
   ace_mode: text
   language_id: 231
+MiniZinc:
+  type: programming
+  color: "#1295e6"
+  extensions:
+  - ".mzn"
+  - ".dzn"
```

Comment:
Good idea! I've added a separate "MiniZinc Data" language and set it to not be `searchable`.